components/RootHomePage/Keynotes.js

```diff
@@ -1,12 +1,92 @@
 import React from 'react';
 import styled from 'styled-components';
+import { Grid, Cell } from 'styled-css-grid';
+import YouTubeVideo from '../shared/YouTubeVideo';
+
+import { gridRepeat, below } from '../../utilities';
+import ContainerWithBGImageAndLG from './ContainerWithBGImageAndLG';
+
 import ContentSection from '../shared/ContentSection';
 
+const Container = styled.div``;
+const Content = styled(ContentSection)`
+  text-align: center;
+
+  h1 {
+    margin-bottom: 5rem;
+  }
+`;
+
+const StyledGrid = styled(Grid)`
+  ${below.small`
+    display: flex;
+    flex-direction: column;
+  `};
+`;
+
+const Title = styled.div`
+  font-weight: bold;
+  line-height: 1.4;
+`;
+
+const Counselor = styled.p`
+  margin: 0;
+  font-size: 1.3rem;
+  margin-bottom: 1rem;
+`;
+
+const ResponseiveYouTube = styled(YouTubeVideo)`
+  width: 35rem;
+  height: 20rem;
+  margin: auto;
+
+  ${below.small`
+    width: 25rem;
+    height: 15rem;
+  `};
+`;
+
+const YouTube = ({ videoId }) => {
+  return (
+    <ResponseiveYouTube
+      videoId={videoId}
+      autoplay={0}
+      rel={0}
+      modest={1}
+      containerHeight="35rem"
+      containerWidth="20rem"
+    />
+  );
+};
+
 const Keynotes = ({ className }) => {
   return (
-    <ContentSection className={className}>
-      <p>this is root keynote section</p>
-    </ContentSection>
+    <Container className={className}>
+      <ContainerWithBGImageAndLG
```

Comment:
Can you use `ContentSection` for this now?

Reply:
This is weird section where only the top piece has a full-width image.